sdk/core/azure-core/azure/core/settings.py

```diff
@@ -110,6 +112,47 @@ def convert_logging(value):
     return level
 
 
+def get_opencensus_wrapper():
+    # type: () -> OpencensusWrapper
+    """Returns the OpencensusWrapper if opencensus is installed else returns None"""
+    try:
+        from azure.core.tracing.ext.opencensus_wrapper import OpencensusWrapper
+
+        return OpencensusWrapper
+    except ImportError:
+        return None
+
+
+def convert_tracing_impl(value):
+    # type: (Union[str, AbstractSpan]) -> AbstractSpan
+    """Convert a string to AbstractSpan
+
+    If a AbstractSpan is passed in, it is returned as-is. Otherwise the function
+    understands the following strings, ignoring case:
+
+    * "opencensus"
+
+    :param value: the value to convert
+    :type value: string
+    :returns: AbstractSpan
+    :raises ValueError: If conversion to AbstractSpan fails
+
+    """
+    if issubclass(value.__class__, AbstractSpan) or value is None:
+        return value
+
+    _impl_dict = {"opencensus": get_opencensus_wrapper()}
```

Comment:
This always invokes `get_opencensus_wrapper`.

Reply:
moved the dict out of the function